README.md

```diff
@@ -141,6 +142,19 @@ If you use [MacPorts](https://www.macports.org) you can install like this:
 
 ### Linux
 
+#### Homebrew
```

Comment:
Do you mind rearranging this section as:

- Debian
- Arch Linux
- Homebrew
- Manual installation

Homebrew is definitely not the #1 pkg mgr on Linux :D